### PR TITLE
Fix: Display user reputation in its non-scientific notation format

### LINKED ISSUE #2706: User popover shows reputation in wrong units
## Steps to reproduce

1. Click on any member with reputation to open the popover.
2. Hover over the reputation percentage value in any team.

image

## Expected behaviour

Value is shown in points, such as here in the user hub:
image

## Actual behaviour

Reputation gets multiplied by 10^18.

### PR DESCRIPTION
**Pull request by rumzledz**
## Description

| Before | After |
| ---- | ---- |
| image | image |

## Testing

1. Visit the Members page
2. Click on leela, the User Info popover should come up
3. Hover on the Reputation percentage
4. Verify that reputation value in the tooltip is not being shown in scientific notation anymore

Resolves #2706

### CODE REVIEW

File: src/components/v5/shared/UserInfoPopover/partials/UserInfo.tsx [hunk 1]
```diff
@@ -134,7 +138,14 @@ const UserInfo: FC<UserInfoProps> = ({
...
                           className="flex min-w-[4.5rem] items-center justify-end text-sm text-blue-400"
                           tooltipContent={
-                            <Numeral value={reputationRaw} suffix=" pts" />
+                            <Numeral
+                              value={reputationRaw}
+                              suffix=" pts"
+                              decimals={getTokenDecimalsWithFallback(
+                                colony.nativeToken.decimals,
+                                DEFAULT_TOKEN_DECIMALS,
+                              )}
```
Comment (mmioana) [inline src/components/v5/shared/UserInfoPopover/partials/UserInfo.tsx:147]: Given the `getTokenDecimalsWithFallback` already uses `DEFAULT_TOKEN_DECIMALS` as it's fallback value I think we can skip passing it as the 2nd argument

**Review by mmioana — APPROVED** [review]
Really cool you picked this up @rumzledz 🤩 
Now the tooltip value shows up nicely
Screenshot 2024-08-14 at 09 30 22

Only left a small comment if you consider to pick it up 🙏

File: src/components/v5/shared/UserInfoPopover/partials/UserInfo.tsx (hunk 1 above)
Comment (rumzledz) [reply src/components/v5/shared/UserInfoPopover/partials/UserInfo.tsx:147]: Oh you're right @mmioana , you know what I'll clean up other places where this is being done as well 👌

Comment (rumzledz) [reply src/components/v5/shared/UserInfoPopover/partials/UserInfo.tsx]: @mmioana pushed! 🚀  I've also cleaned up other areas that use this redundant 2nd argument

**Review by bassgeta — APPROVED** [review]
Nice small fix and a nice cleanup of the redundant fallback, good job :muscle: :triumph: 
image
Awarding a reputation shows up nicely
image
:ship:

**Review by davecreaser — APPROVED** [review]
Nice fix, nice clean up, nice job @rumzledz! 🧹 

Working nicely:
Screenshot 2024-08-14 at 14 21 16

**Review by Nortsova — APPROVED** [review]
Nice job @rumzledz ! Thanks for clean up

Reputation value showed up correctly:
image

**Pull request merged**